scripts/exrrfs_analysis_enkf.sh

```diff
@@ -115,6 +114,15 @@ l_fv3reg_filecombined=.false.
 #-----------------------------------------------------------------------
 #
 fixgriddir=${FIX_GSI}/${PREDEF_GRID_NAME}
+
+# DELETE THESE LINES
+#if [ "${CYCLE_TYPE}" = "spinup" ]; then
+#   enkfanal_nwges_dir="${GESROOT}/${RUN}.${PDY}/${cyc}_spinup/anal_enkf_spinup"
+#else
+#   enkfanal_nwges_dir="${GESROOT}/${RUN}.${PDY}/${cyc}/anal_enkf"
+#fi
+#mkdir -p ${enkfanal_nwges_dir}
+
 cpreq -p ${fixgriddir}/fv3_coupler.res    coupler.res
```

Comment:
I think these lines should be deleted since they aren't needed anymore.

Reply:
Yes. They can be removed.